From b8f2c1edad1684a4d4c0f30bc779f67707be5ae7 Mon Sep 17 00:00:00 2001
From: Yao Zi <ziyao@disroot.org>
Date: Sat, 30 Aug 2025 10:51:05 +0000
Subject: [PATCH 344/347] AOSCOS: PCI: Add vendor ID for Motorcomm Electronic
 Technology

This company produces Ethernet controllers and PHYs. Add their vendor
ID, 0x1f0a[1], which is recorded by PCI-SIG and has been seen on their
PCI Ethernet cards.

Link: https://pcisig.com/membership/member-companies?combine=1f0a # [1]
Signed-off-by: Yao Zi <ziyao@disroot.org>
---
 include/linux/pci_ids.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index 0fb1a1b8954c..55d6df03b393 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -2632,6 +2632,8 @@
 
 #define PCI_VENDOR_ID_CXL		0x1e98
 
+#define PCI_VENDOR_ID_MOTORCOMM		0x1f0a
+
 #define PCI_VENDOR_ID_TEHUTI		0x1fc9
 #define PCI_DEVICE_ID_TEHUTI_3009	0x3009
 #define PCI_DEVICE_ID_TEHUTI_3010	0x3010
-- 
2.51.0

